app/views/hyrax/admin/features/index.html.erb

```diff
@@ -0,0 +1,70 @@
+<% #OVERRIDE HYRAX 3.5.0 to skip show_featured_works for the dc tenant %>
+<% provide :page_header do %>
+  <h1><span class="fa fa-wrench" aria-hidden="true"></span> <%= t('.header') %></h1>
+<% end %>
+<div class="flip row">
+  <div class="col-md-12">
+    <div class="card">
+      <div class="card-body">
+        <div class="table-responsive">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th></th>
+                <th class="name"><%= t('.feature') %></th>
+                <th class="description"><%= t('.description') %></th>
+                <th class="action"><%= t('.action') %></th>
+              </tr>
+            </thead>
+            <tbody>
+            <% @feature_set.grouped_features.each do |group, features| -%>
+              <% if @feature_set.grouped? -%>
+                <tr class="group">
+                  <td></td>
+                  <td class="name" colspan="<%= 2 + @feature_set.strategies.size -%>">
+                    <h2>
+                      <%= t(group ? group.name : :default, scope: [:flipflop, :groups], default: group ? group.title : nil) -%>
+                    </h2>
+                  </td>
+                </tr>
+              <% end -%>
+              <% #OVERRIDE HYRAX 3.5.0 to skip show_featured_works per denylist: TENANTS_WITH_NO_WORK_FEATURES %>
+              <% modified_features = features.reject { |item| item.instance_variable_get(:@key) == :show_featured_works && AccountsHelper::TENANTS_WITH_NO_WORK_FEATURES.include?(current_account.name) } %>
+              <% modified_features.each do |feature| %>
```

Comment:
Can you talk to me about what the item looks like?  It's a little suspect to use `instance_variable_get` as that's rather fragile.

Also, instead of looping through the features to reject, then looping again, consider moving to

```
<% features.each do |feature| %>
<% next if feature.instance_variable_get(:@key) == :show_featured_works && AccountsHelper::TENANTS_WITH_NO_WORK_FEATURES.include?(current_account.name) %>
```

This reduces the looping, which may be beneficial.